src/core/Akka.Remote.Tests.Performance/Transports/TestTransportRemoteMessagingThroughputSpec.cs

```diff
@@ -1,9 +1,13 @@
+using System;
 using Akka.Configuration;
+using Akka.Remote.Transport;
 
-namespace Akka.Remote.Tests.Performance
+namespace Akka.Remote.Tests.Performance.Transports
 {
     public class TestTransportRemoteMessagingThroughputSpec : RemoteMessagingThroughputSpecBase
     {
+        private readonly string _registryKey = Guid.NewGuid().ToString();
```

Comment:
Failing to do this for every individual benchmark run is what caused #1547 and the build failure in #1550. Essentially the `AssociationRegistry` object used by `TestTransport` has static members that retain all of the address and connection information for each spec, and we were re-using the same `AssociationRegistry` each time instead of getting a clean instance. So this gives us a new `registry-key` each time and thus ensures a clean `AssociationRegistry` instance also.